src/fixup/fixup.cpp

```diff
@@ -717,6 +717,13 @@ TaskStatus ApplyFloors(T *rc) {
 }
 
 template TaskStatus ApplyFloors<MeshBlockData<Real>>(MeshBlockData<Real> *rc);
+template <>
+TaskStatus ApplyFloors<MeshData<Real>>(MeshData<Real> *rc) {
+  for (int b = 0; b < rc->NumBlocks(); b++) {
+    ApplyFloors(rc->GetBlockData(b).get());
```

Comment:
Does `GetBlockData(b)` definitely return the right stage? it should I think? Might be worth double checking the name that gets spit out here